joomla/joomla-cms
# IIS - Add woof mime type for media folder in web.config.txt

This PR add woof mime type for iis web.config because it is not configured by default. This causes 404 errors on files of type ".woff ". Only configure this for media folder.
## Actual result in "Article Manager: Edit Article"

![2015-06-13 13_00_40-joomla-cms - administration - article manager_ edit article](https://cloud.githubusercontent.com/assets/5129879/8144093/e893a132-11cb-11e5-8d89-770d379575c2.jpg)
## Actual result in "   Banner Manager Options"

![2015-06-13 13_10_41-joomla-cms - administration - banner manager options](https://cloud.githubusercontent.com/assets/5129879/8144119/580c321c-11cd-11e5-9ae4-6ab8b28af8f3.jpg)
## Expected result

No 404 errors
## How to test

You must use IIS for test this PR
1. In administration, edit article,
2. You must see 404 errors,
3. Apply the patch and refesh the page,
4. There are more error.
